test/suites/expiration/expiration_test.go

```diff
@@ -12,9 +12,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package integration_test
+package deprovisioning_test
```

Comment:
nit: Is there a reason we wouldn't call this `expiration_test`? Looks like the drift suite package is `drift`, which it should probably either be `deprovisioning_test` or `drift_test` to be aligned.

Reply:
Whoops, just a miss. There is no reason